notebook/static/base/js/security.js

```diff
@@ -31,6 +31,10 @@ define([
                 }
             }
         }
+        // TODO(julienr): This is a ugly hack to work around the fact that
+        // by default caja doesn't seem to like base64 src for inline markdown
+        // images. Not sure if we should do that here
+        ATTRIBS['img::src'] = 0;
```

Comment:
After a bit of reading, I [believe](https://www.owasp.org/index.php/Script_in_IMG_tags) this is not an issue on any supported browser. Can you make a note here that img:src is safe on browsers post ie6?